doc/security/reporting.rst

```diff
@@ -180,7 +182,12 @@ following releases:
 The developer of the fix shall be responsible for any necessary
 backports, and apply them to any of the above listed release branches,
 unless the fix does not apply (the vulnerability was introduced after
-this release was made).
+this release was made). All recommendations for
+:ref:`vulnerability fixes <vulnerability_fix_recommendation>` apply
+for backport pull request (and associated issue). Additionally is
```

Comment:
pull requests, associated issues...

Additionally, it is recommended

Reply:
done